Dockerfile.ubuntu

```diff
@@ -45,14 +41,18 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
             pkg-config \
             wget \
             zlib1g-dev && \
+            update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 9 && \
+            update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 9 && \
             apt-get clean && \
             rm -rf /var/lib/apt/lists/*
 
 ####### Azure SDK needs new boost:
 WORKDIR /boost
 # hadolint ignore=DL3003
 RUN wget -nv https://sourceforge.net/projects/boost/files/boost/1.69.0/boost_1_69_0.tar.gz && \
-tar xvf boost_1_69_0.tar.gz && cd boost_1_69_0 && ./bootstrap.sh && \
+tar xf boost_1_69_0.tar.gz && cd boost_1_69_0 && ./bootstrap.sh && \
+sed -i -e 's|#if PTHREAD_STACK_MIN > 0|#ifdef PTHREAD_STACK_MIN|g' boost/thread/pthread/thread_data.hpp && \
+#https://github.com/boostorg/thread/pull/297/files
```

Comment:
What does this line do?

Reply:
Can we add such short explanation along with the link to github PR? It would be easier to pick it up in the future when we revisit those lines for some reason.